dev/dev.clj

```diff
@@ -17,18 +17,18 @@
    [com.stuartsierra.component :as component]
    [com.stuartsierra.component.repl :refer [set-init]]
    [modular.postgres]
-   [background-processing.background-processor :as background-processor]
-   [background-processing.enqueuer :as enqueuer]
+   #_ [background-processing.background-processor :as background-processor]
+   #_ [background-processing.enqueuer :as enqueuer]
    [playground.server]
    [playground.service]))
 
 (defn dev-system
   []
   (component/system-map
    :service-map playground.server/dev-map
-   :background-processor (background-processor/new :queue-name "cljtest")
-   :enqueuer (enqueuer/new :queue-name "cljtest")
-   :db (modular.postgres/map->Postgres {:url "jdbc:postgresql:ebdb" :user "vemv" :password ""})
+   ;; :background-processor (background-processor/new :queue-name "cljtest")
+   ;; :enqueuer (enqueuer/new :queue-name "cljtest")
+   :db (modular.postgres/map->Postgres {:url "jdbc:postgresql:swallows" :user "swallows" :password "swallows"})
```

Comment:
Doesn't work with this :db map (tried long address same result) 
looked into the modular.postgres ns. I don't understand how pooling work. (related?)